invenio_communities/config.py

```diff
@@ -26,6 +26,7 @@
     "upload": "/communities/<pid_value>/upload",
     "settings": "/communities/<pid_value>/settings",
     "requests": "/communities/<pid_value>/requests",
+    "subcommunities": "/communities/<pid_value>/browse/subcommunities",
```

Comment:
Shouldn't this be as defined below by the resource? 

e.g. 
`/communities/<pid_value>/subcommunities`

The concept of `browse` does not exist in `invenio-communities` at this point